python/aqora_cli/qiskit/client.py

```
@@ -0,0 +1,290 @@
+    try:
+        asyncio.get_running_loop()
+    except RuntimeError:
+        return asyncio.run(run_factory())
+
+    result: dict[str, Any] = {}
+    error: dict[str, BaseException] = {}
+
+    def runner() -> None:
+        try:
+            result["value"] = asyncio.run(run_factory())
+        except BaseException as exc:  # pragma: no cover - thread bridge
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `BaseException`

(BLE001)